### PR TITLE
[docs][yba] Migrate YBA from Replicated

### PR DESCRIPTION
**Pull request by ddhodge**
Using YBA Installer to migrate installation from Replicated
PLAT-10754

@netlify /preview/yugabyte-platform/install-yugabyte-platform/

### CODE REVIEW

**Review by shubin-yb — CHANGES_REQUESTED** [review]

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 1]
```diff
@@ -216,6 +220,54 @@ INFO[2023-04-24T23:19:59Z] Successfully installed YugabyteDB Anywhere!
...
+    $ sudo ./yba-ctl replicated-migrate start -l /path/to/license
+    ```
+
+    The `start` command runs all [preflight checks](#run-preflight-checks) and then proceeds to do the migration, and then waits for YBA to start. After the migration succeeds, you can immediately start using YBA.
+
+1. Validate YBA is up and running with the correct data, including Prometheus.
+
+    At this point, if you find problems, you can revert to your Replicated installation using the `replicated-migrate rollback` command. Note that any changes made with the new YBA (either using the UI or the API) will not be reflected after the rollback.
+
+1. If the new YBA installation is correct, finish the migration as follows:
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:263]: @shubin-yb It says "At this point, if you find problems, you can revert to your Replicated installation..."

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 2]
```diff
@@ -216,6 +220,76 @@ INFO[2023-04-24T23:19:59Z] Successfully installed YugabyteDB Anywhere!
...
+
+{{< /note >}}
+
+Review the [prerequisites](#prerequisites). YBA Installer performs the migration in place. Make sure you have enough disk space on your current machine.
+
+### Migrate a YBA installation
+
+To migrate your installation from Replicated, do the following:
+
+1. If your Replicated installation is v2.18.5 or earlier, [upgrade your installation](../../../upgrade/upgrade-yp-replicated/) to v2.20.1.
```
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:239]: cc @shubin-yb
Ya that makes it more complicated
"If your Replicated installation is v2.18.5 or earlier, or v2.20.0, upgrade..." - clear enough?

[the remark this answers is not in the document]
Comment (chidmuthu) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:247]: Yes, I think that makes sense!

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 3]
```diff
@@ -216,6 +220,76 @@ INFO[2023-04-24T23:19:59Z] Successfully installed YugabyteDB Anywhere!
...
+
+1. If your instances are v2.18.5 or earlier, [upgrade your primary and HA standby instances](../../../administer-yugabyte-platform/high-availability/#upgrade-instances) to v2.20.1.
+1. Migrate and finalize the primary.
+1. Migrate and finalize the standby.
+
+Failovers are now possible again after the completion of this step.
+
+If Replicated is using HTTP:
+
+1. [Remove the standby instances](../../administer-yugabyte-platform/high-availability/#remove-a-standby-instance).
```
[the remark this answers is not in the document]
Comment (chidmuthu) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md]: Yes, they also have to delete the entire configuration, not just the instances.

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (shubin-yb) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:247]: I think this makes sense

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (chidmuthu) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md]: There's a `Delete Configuration` button in the top right of the Admin HA configuration section

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:271]: Note in intro above, and new section on HA below...

[the remark this answers is not in the document]
Comment (ddhodge) [reply docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:273]: OK added bit about not configuring HA before `finish`...

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 4]
```diff
@@ -216,6 +220,87 @@ INFO[2023-04-24T23:19:59Z] Successfully installed YugabyteDB Anywhere!
...
+1. If the new YBA installation is correct, finish the migration as follows:
+
+    ```sh
+    $ sudo ./yba-ctl replicated-migrate finish
+    ```
+
+    This uninstalls Replicated and makes the new YBA instance permanent.
+
+### Migration and high availability
+
```
Comment (chidmuthu) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:282]: As a catchall let's highlight that failovers are NOT possible until after finalization  before we go into HTTP/HTTPS.

**Review by shubin-yb — APPROVED** [review]
LG, thanks for iterating on this!

File: docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md [hunk 5]
```diff
@@ -258,7 +343,11 @@ Services:
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer.md:343]: just minor that can be addressed later- the output of the status command is not rendered clearly : https://deploy-preview-20043--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/install-software/installer/#service-management

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-replicated.md [hunk 6]
```diff
@@ -11,6 +11,14 @@ menu:
 type: docs
 ---
 
+{{< note title="Replicated end of life" >}}
+
+YugabyteDB Anywhere will end support for Replicated installation at the end of 2024. You can migrate existing Replicated YugabyteDB Anywhere installations using YBA Installer. See [Migrate from Replicated](../installer/#migrate-from-replicated).
```
Comment (aishwarya24) [inline docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-replicated.md:16]: broken link.

**Review by aishwarya24 — APPROVED** [review]
perform the same changes to stable.

**Pull request merged**